src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Resolver/ParameterResolverTest.php

```diff
@@ -245,10 +262,94 @@ public function testResolveWithoutUrlsParameter()
                     'url' => '/de/test',
+                    'language' => 'de',
+                    'country' => null,
+                ],
+            ],
+            'webspaceKey' => 'sulu',
+            'segments' => [],
+        ], $resolvedData);
+    }
+
+    public function testResolveWithoutUrlsAndCountryParameter(): void
+    {
+        $parameterResolver = new ParameterResolver(
+            $this->structureResolver->reveal(),
+            $this->requestAnalyzerResolver->reveal(),
+            $this->webspaceManager->reveal(),
+            $this->requestStack->reveal(),
+            '_sulu_segment_switch',
+            ['urls' => false]
+        );
+
+        $localization1 = $this->prophesize(Localization::class);
+        $localization1->getLocale()->willReturn('en_gb');
+        $localization1->getLanguage()->willReturn('en');
+        $localization1->getCountry()->willReturn('gb');
+        $localization2 = $this->prophesize(Localization::class);
+        $localization2->getLocale()->willReturn('de_at');
+        $localization2->getLanguage()->willReturn('de');
+        $localization2->getCountry()->willReturn('at');
+
+        $this->structureResolver->resolve($this->structure->reveal(), true)
+            ->shouldBeCalledTimes(1)
+            ->willReturn([
+                'content' => [],
+                'view' => [],
+                'urls' => [
+                    'en_gb' => '/test',
+                    'de_at' => '/test',
+                ],
+                'extension' => [
+                    'seo' => [],
+                    'excerpt' => [],
+                ],
+            ]);
+        $this->requestAnalyzerResolver->resolve($this->requestAnalyzer)
+            ->shouldBeCalledTimes(1)
+            ->willReturn(['webspaceKey' => 'sulu']);
+        $this->webspaceManager->findUrlByResourceLocator('/test', null, 'en_gb')->willReturn('/en/test');
+        $this->webspaceManager->findUrlByResourceLocator('/test', null, 'de_at')->willReturn('/de/test');
+        $this->requestAnalyzer->getPortal()->willReturn($this->portal->reveal())->shouldBeCalledTimes(1);
+        $this->portal->getLocalizations()
+            ->willReturn([$localization1->reveal(), $localization2->reveal()])->shouldBeCalledTimes(1);
+
+        $this->webspace->getSegments()->willReturn([]);
+
+        $resolvedData = $parameterResolver->resolve(
+            [
+                'testKey' => 'testValue',
+            ],
+            $this->requestAnalyzer->reveal(),
+            $this->structure->reveal()
+        );
+
+        $this->assertEquals([
+            'testKey' => 'testValue',
+            'content' => [],
+            'view' => [],
+            'extension' => [
+                'seo' => [],
+                'excerpt' => [],
+            ],
+            'localizations' => [
+                'en_gb' => [
+                    'locale' => 'en_gb',
+                    'url' => '/en/test',
+                    'language' => 'en',
```

Comment:
I discussed this with @nnatter I would not add the `language` at current state. I'm afraid that somebody use the `language` instead of `locale` in `sulu_content_path` and then they are not sure why it don't work like expected. We had in the past a lot of confusen with path and url which where also very similar and a lot did fall into this pitfall so alson as there is no really need for the language I would avoid creating this pitfall here.

Sorry for inconveniences!

Reply:
@plozmun i am sorry - i did not think about this 🙈 i think it makes sense to no add the `language` to avoid confusion

Reply:
Ok! no problem. Maybe we can change these two parameters to `iso_language` and `iso_region` witch defines better the ISO specification. WDYT?

Reply:
Actually we don't know if they did input a iso languages or iso countries its not validated and sadly did see projects using none iso strings in there localization, so I would just stay with `country` here at current state.

Reply:
Ok, language removed :)